.github/workflows/publish-chart.yaml

```diff
@@ -4,6 +4,11 @@ name: Publish Helm chart
 on:
   workflow_dispatch:
   pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - closed
     branches:
       - main
     paths:
```

Comment:
Should helm tests be started if this workflow has been changed?

Reply:
I'm unsure about it. in any case we can see the result for created PR.